NachoSpigot-Server/src/main/java/net/techcable/tacospigot/HopperPusher.java

```diff
@@ -1,15 +1,10 @@
 package net.techcable.tacospigot;
 
-import net.minecraft.server.AxisAlignedBB;
-import net.minecraft.server.BlockPosition;
-import net.minecraft.server.Entity;
-import net.minecraft.server.MathHelper;
-import net.minecraft.server.TileEntityHopper;
-import net.minecraft.server.World;
+import net.minecraft.server.*;
```

Comment:
is it that good to cover them all with *?

Reply:
I believe that it's better practice to do .* if u have 5 or more imports of the same directory

Reply:
normally i would say yes but net.minecraft.server is a bunch of classes. or is that irrelevant for compiling?

Reply:
What's ur point on "net.minecraft.server is a bunch of classes"

Reply:
that it takes a bit more time to go through the classes, but i think thats irrelevant, right?

Reply:
As far as I know it affects nothing performance wise

Reply:
Java byte code uses full class names not imports, so it wouldn’t affect performance